jboss-eap-cd-openshift-modules/module.yaml

```diff
@@ -7,7 +7,7 @@ execute:
   user: '185'
 
 artifacts:
-- path: txn-recovery-marker-jdbc-common-1.1.2.Final-redhat-00001.jar
+- name: txn-recovery-marker-jdbc-common-1.1.2.Final-redhat-00001.jar
```

Comment:
I suggest to use something like this:
```yaml
- name: txn-recovery-marker-jdbc-common
  target: txn-recovery-marker-jdbc-common-1.1.2.Final-redhat-00001.jar
  md5: cd68ad886a759d21f8dd0cb7646601ee

Same applies to other artifacts.
```

Reply:
Ah, I see, so it can be overriden via name. Clever :)

Reply:
Exactly!